dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Rules/ParameterCount/ParameterCountAnalyzer.cs

```diff
@@ -147,11 +153,27 @@ private static string GetNameForMethod(IMethodSymbol method)
 
         private static void AnalyzeParameters(SymbolAnalysisContext context, ParameterCountInfo parameterCountInfo, string memberName)
         {
-            ImmutableArray<IParameterSymbol> parameters = parameterCountInfo.MethodSymbol.Parameters;
+            var parameters = parameterCountInfo.MethodSymbol.Parameters;
+
+            var excludedParameters = parameters.AsEnumerable()
```

Comment:
not sure if this is quite the right name for it, as it seems to be the parameters _without_ the exclusions?